src/CoreWCF.Primitives/src/CoreWCF.Primitives.csproj

```diff
@@ -22,6 +22,7 @@
     <PackageReference Include="System.Security.Principal.Windows" Version="5.0.0" />
     <PackageReference Include="System.Reflection.DispatchProxy" Version="4.7.1" />
     <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.4" />
+    <PackageReference Include="System.Web.Services.Description" Version="4.9.0" />
```

Comment:
does this have any issues across platforms ?

Reply:
None. This works on all platforms as it targets netstandard2.0 and type forwards on .NET Framework to system.web.services.dll in the GAC.